vunit/vivado/vivado.py

```diff
@@ -30,8 +36,9 @@ def add_from_compile_order_file(
 
     no_dependency_scan = []
     with_dependency_scan = []
+    verilog_file_endings = (".v", ".vp", ".sv")
     for library_name, file_name in compile_order:
-        is_verilog = file_name.endswith(".v") or file_name.endswith(".vp")
+        is_verilog = file_name.endswith(verilog_file_endings)
```

Comment:
Rather than using `.endswith`, I would suggest using Pathlib's `.suffix` (https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix) because semantically we are cheking the extension of a filename not the end of an arbitrary string. At the same time, since the tuple/list is used once only, the declaraction of the variable feels redundant. I would suggest `Path(file_name).suffix in ['.v', '.vp', '.sv']`.

Reply:
Sure.

Reply:
Part of fixing #796